File: 03-Code/OCAppUsingSwift/OCAppUsingSwift/ViewController.m
//
//  ViewController.m
//  OCAppUsingSwift
//
//  Created by min Lee  on 2024/3/20.
//  Copyright © 2024 min Lee. All rights reserved.
//

#import "ViewController.h"
#import "OCAppUsingSwift-Swift.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    
    //OC中使用Swift
    Person *p = [[Person alloc] initWithName:@"哈哈" age:3];
    [p getInfo];
    [Person staticMethod];
    [Person classMethod];
}

@end

File: 03-Code/SwiftAppUsingOC/SwiftAppUsingOC/ViewController.swift
//
//  ViewController.swift
//  SwiftAppUsingOC
//
//  Created by min Lee  on 2024/3/20.
//  Copyright © 2024 min Lee. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
        let obj = OCTest();
        obj.test();
        
        OCTest.test();
        
        obj.ocCallSwift();
    }
}

